doc/nrf/app_dev/bootloaders: update about Mcuboot's LOAD_ADDRESS_CHECK (#29857)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf54l/ecies_x25519.rst

  This is the default configuration.
* Encryption is not supported when using MCUboot in direct-xip mode.
* Storing the ECIES-X25519 device private key in the Key Management Unit (KMU) is currently not supported.

> * Encryption of MCUboot updates can be supported with an immutable and upgradable bootloader configuration, with the following limitations:

Reviewer: ```suggestion
* Encryption of MCUboot updates is supported with an immutable and upgradable bootloader configuration, with the following limitations:
```

>   * The active instance of MCUboot must use image matching based on explicit image address (:kconfig:option:`CONFIG_MCUBOOT_CHECK_HEADER_LOAD_ADDRESS=y`).

Reviewer 2: ```suggestion
  * The active instance of MCUboot must use image matching based on explicit image address (:kconfig:option:`CONFIG_MCUBOOT_CHECK_HEADER_LOAD_ADDRESS=y`) or vendor and class UUID-based matching.
```

>   * The same key encryption key must be used for both MCUboot updates and the application image.

Reviewer: ```suggestion
  * You must use the same key encryption for both MCUboot updates and the application image.
```

>   * The image must be signed manually, since the NCS signing process for s0/s1 images has not yet been updated to support encryption.

Reviewer: ```suggestion
  * You must sign the image manually, since the NCS signing process for s0/s1 images has not yet been updated to support encryption.
```
